File: MODUL 5 Praktikum Pemrograman/PRAKTIKUM 5 C/PRAK505-2410817120021-Aulia-Az-Zahra.c
#include <stdio.h>
void Biodata(int lahir, char nama[], char asal[])
{
    //Lengkapi Function ini
    int tahun_Sekarang = 2020;
    int umur = tahun_Sekarang - lahir;
    printf("Perkenalkan Nama Saya : %s", nama);
    printf("\nUmur Saya : %d", umur);
    printf("\nSaya Adalah Angkatan : %d", tahun_Sekarang);
    printf("\nAsal Saya dari : %s", asal);
}
int main()
{
    int tahunLahir;
    char Namaku[20], Asal[15];
    scanf(" %d",&tahunLahir);
    scanf(" %[^\n]%*c",&Namaku);
    scanf(" %[^\n]%*c",&Asal);
    Biodata(tahunLahir, Namaku, Asal);
    return 0;
} 

File: MODUL 5 Praktikum Pemrograman/PRAKTIKUM 5 PY/PRAK505-2410817120021-Aulia-Az-Zahra.py
def Biodata(lahir, nama, asal):
    #Lengkapi Function ini
    tahun_sekarang = 2020
    umur = tahun_sekarang - lahir
    print(f"Perkenalkan Nama Saya : {nama}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal}")

tahunLahir = int(input())
Namaku = input()
Asal = input()
Biodata(tahunLahir, Namaku, Asal)